src/client/components/measure-list/measure-list.vue

```diff
@@ -70,15 +71,36 @@ export default {
         : this.measures
     },
     filteredMeasures() {
-      return this.sortedMeasures.filter(measure => {
+      const types = {
+        'Polygon': 'canDrawPolygon',
+        'LineString': 'canDrawPolyline',
+        'Point': 'canDrawPoint',
+      }
+      const type = this.selectedType();
+      const availableMeasures = type
+        ? this.filterMeasureByType(types[type])
```

Comment:
`type` refers to the currently selected type. On L79 you reference it from `selectedType`. Because we know what type we are dealing with (the _selected_ type) we can make the statement `types[type]` more explicit by doing: `types[selectedType]`. This way we indicate that from the `types` array we want the current selected type, not some random type.

tldr; `types[type]` => `types[selectedType]`